spyderlib/spyder.py

```diff
@@ -542,20 +542,24 @@ def setup(self):
                                         context=Qt.ApplicationShortcut)
             self.register_shortcut(self.close_dockwidget_action, "_",
                                    "Close pane")
-
+            self.lock_dockwidgets_action = create_action(self, _("Lock panes"),
+                                            toggled=self.toggle_lock_dockwidgets,
+                                            context=Qt.ApplicationShortcut)
+            self.register_shortcut(self.lock_dockwidgets_action, "_",
+                                       "Lock panes")

```

Comment:
There is alwaya problem with changing text: is the displayed text the status or the action to perform ? A checkbox is much more obvious.